packages/component/src/decorator/private/Decorator.tsx

```diff
@@ -1,8 +1,10 @@
 import { DecoratorComposer, type DecoratorMiddleware } from 'botframework-webchat-api/decorator';
```

Comment:
It is optional and exported. I think it may be considered a breaking change to include it. I marked it as experimental. I'd expect we to not rush and include it by default maybe in 4.20.x